application-mocca-calendar-ui/src/main/resources/MoccaCalendar/WebHome.xml

```diff
@@ -70,6 +72,23 @@
 })
 #set ($columns = ['doc.title', 'doc.creator', '_actions'])
 #livetable('moccacalendar' $columns $columnsProperties $options)
+(% style="border-color:gray" %)
+----
+====== $escapetool.xml($services.localization.render('MoccaCalendar.calendar.viewSubscribed')) ======
+## Subscribed calendars livedata
+#set ($columns = ['doc.title', 'doc.creator', '_actions'])
+#set ($options = {
+  'className': 'MoccaCalendar.MoccaCalendarSubscribeClass',
+  'resultPage': 'MoccaCalendar.Code.MoccaCalendarLiveTableResults',
+  'translationPrefix': 'moccacalendar.livetable.',
+  'tagCloud': false,
+  'rowCount': 15,
+  'maxPages': 10,
+  'selectedColumn': 'doc.subscribed',
+  'defaultOrder': 'asc'
+})
+#livetable('subscribedCalendars' $columns $columnsProperties $options)
```

Comment:
You should opt for a livedata macro instead of a livetable.
https://extensions.xwiki.org/xwiki/bin/view/Extension/Live%20Data%20Macro/

Reply:
I've removed the CSS to have the default color for the separator.